test/common/network/proxy_protocol_test.cc

```diff
@@ -53,7 +54,11 @@ class ProxyProtocolTest : public testing::Test {
   std::shared_ptr<MockReadFilter> read_filter_;
 };
 
-TEST_F(ProxyProtocolTest, Basic) {
+// Parameterize the listener socket address version.
+INSTANTIATE_TEST_CASE_P(IpVersions, ProxyProtocolTest,
+                        testing::ValuesIn(TestEnvironment::getIpVersionsForTest()));
```

Comment:
Does this test benefit from IP versions parameterization? Seems both IPv4 and IPv6 tests can be run given there is no loopback dependency.